bridge/docs/bridging.md

```diff
@@ -34,3 +34,91 @@ Again, a withdrawfee of 1 TFT will be taken, so make sure you send a larger amou
+
+3. The bridge validators are listening for this event. They extract the `burnId` and other transaction parameters, validate the Stellar address (tokens could refunded/minted back on TFChain at this step if validation failed), then construct signed stellar transaction and extract the signature (note, the transaction can not be submitted yet to stellar network). They then call `propose_burn_transaction_or_add_sig` extrinsic which fill their signatures and the bridge account sequence number in the `BurnTransaction` in storage that matches specified `burnId`. when this call executed, the `BurnTransactionSignatureAdded` event is emitted. 
+
+4. If the majority (more than the half) of bridge validators provided their signature for a transaction, a `BurnTransactionReady` event is emitted as well. This check happens every time the `propose_burn_transaction_or_add_sig` extrinsic is executed by validator call.
+
+5. The bridge will handle the event and query TFChain storage for the `BurnTransaction` details and the validators’ signatures. It will create a multi-signatures Stellar transaction and submit it to Stellar network. If submitted successfully, it will call `set_burn_transaction_executed` extrinsic (which removes the `BurnTransaction` from the `BurnTransactions` storage and adds it to `ExecutedBurnTransactions`) then emit `BurnTransactionProcessed` event.
+
+#### Overview of the TFChain Burning events
+1. `tftBridgeModule.BurnTransactionCreated`: A swap from TFChain to stellar was initiated by a call to `swap_to_stellar` extrinsic.
+2. `tftBridgeModule.BurnTransactionSignatureAdded`: Other bridge validators handled `BurnTransactionCreated` TFChain event and submit their signature for the proposed stellar transaction.
+3. `tftBridgeModule.BurnTransactionReady`: Enough validators signatures was collected and stored so from now it is possible to submit the proposed stellar withdraw transaction.
+4. `tftBridgeModule.BurnTransactionProcessed`: A bridge validator has called `set_burn_transaction_executed` extrinsic with a proof that the proposed stellar withdraw transaction was executed successfully on stellar network.
+
+#### When a Refund-on-TFChain occurs?
+
+A refund on TFChain is initiated when either of the following conditions is met:
+
+- Account information cannot be retrieved from the Stellar network.
+- The account has no trust line to TFT tokens or has a deleted one (TFT balance limit is `0`).
+
+### TFChain Retry mechanism
+
+We didn't mentioned yet a few TFChain event related to the flows discussed above, these events are:
+- `tftBridgeModule.BurnTransactionExpired`
+- `tftBridgeModule.RefundTransactionExpired`
```

Comment:
- `tftBridgeModule.BurnTransactionExpired:` from TFCHAIN -> STELLAR burn flow, when transaction (burn TFT on TFChain source account and transfer amount from Stellar vault account to Stellar destination account) is stuck in bridge.
- `tftBridgeModule.RefundTransactionExpired:` from STELLAR -> TFCHAIN refund flow, after TFT were locked into Stellar vault, finally not minted to TFChain, and transaction to get it back to Stellar source account is stuck in bridge.